docs/reference/ml/anomaly-detection/apis/get-category.asciidoc

```diff
@@ -48,7 +48,12 @@ include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=job-id-anomaly-detection]
 
 `<category_id>`::
 (Optional, long) Identifier for the category. If you do not specify this
```

Comment:
> If you do not specify this parameter...

I found this sentence a bit long, so I'd probably split it. For example, something like this:

If you specify neither the category ID nor the `partition_field_value`, the API returns information about all categories. If you specify only the `partition_field_value`, the API returns information about all categories for the specified partition.